server/test/services/openApi/v2/Swagger2SchemaGeneratorTest.java

```diff
@@ -0,0 +1,382 @@
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    email:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                sample_enumerator_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    entities:")
+            .appendLine("                      type: array")
+            .appendLine("                      items:")
+            .appendLine("                        type: object")
+            .appendLine("                sample_file_upload_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    file_key:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                    original_file_name:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                sample_id_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    id:")
+            .appendLine("                      type: string")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                sample_number_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    number:")
+            .appendLine("                      type: integer")
+            .appendLine("                      format: int64")
+            .appendLine("                      x-nullable: true")
+            .appendLine("                sample_phone_question:")
+            .appendLine("                  type: object")
+            .appendLine("                  properties:")
+            .appendLine("                    question_type:")
+            .appendLine("                      type: string")
+            .appendLine("                    country_code:")
```

Comment:
In the JSON export, phone number is just a single `phone_number` scalar with the country code prepended